internal/runbits/findproject/project.go

```diff
@@ -20,6 +20,38 @@ func IsLocalProjectDoesNotExistError(err error) bool {
 	return errs.Matches(err, &LocalProjectDoesNotExist{})
 }
 
+func FromInputByPriority(path string, ns *project.Namespaced, cfg projectfile.ConfigGetter, prompt prompt.Prompter) (*project.Project, error) {
+	// Priority #1 - PATH
+	if path != "" {
+		return FromPath(path, ns)
+	}
+
+	// Priority #2 - Namespace
+	if ns != nil {
+		return FromNamespaceLocal(ns, cfg, prompt)
+	}
+
+	// Priority #3 - Env
+	pj, err := project.FromEnv()
```

Comment:
This might be confusing if we ever have a FromEnvVar method. Maybe "FromDisk"? Disregard if I'm misunderstanding, and generally feel free to skip this concern.

Reply:
It seems appropriate to me; an env var is one of the possible sources this function will use.